_posts/2020-05-20-deno-unboxing.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title:  "Deno 1.0"
+author: Contributor
+categories: [ web framework ]
+image: assets/images/deno.png
+tags: [deno, node.js alternative]
+---
+
+#### A Brief Introduction To Deno 1.0
+
+Deno is the creation of Ryan Dahl, which is a new JavaScript runtime for the backend, but instead of being written in C++, it’s written in Rust, based on the Tokio platform (which provides the asynchronous runtime needed by JavaScript), running on Google’s V8 engine. It is also a new runtime for executing JavaScript and TypeScript outside of the web browser. 
+
+
+#### Why should we use this?   
+
+There are some exciting new features in Deno. They are:

```

Comment:
Replace by:
Generally we may get a doubt, why should I use it, here are some of it's features.

Reply:
I will fix all those @puneethkanna

Reply:
Okay @SriSatyaLokesh bro